docs/reference/contrib/settings.rst

```diff
@@ -296,8 +296,8 @@ You could write:
 
 .. code-block:: html+django
 
-    {{ settings.app_label.ImportantPages.page_url.donate_page }}
-    {{ settings.app_label.ImportantPages.page_url.sign_up_page }}
+    {{ settings.app_label.ImportantPages.donate_page.url }}
+    {{ settings.app_label.ImportantPages.sign_up_page.url }}
```

Comment:
Just confirming, I have not run this myself, but should this be `get_url`
https://docs.wagtail.io/en/stable/reference/pages/model_reference.html#wagtail.core.models.Page.get_url
?

Reply:
drive-by comment: I believe `page_url` here relates to https://github.com/wagtail/wagtail/blob/main/wagtail/contrib/settings/models.py#L86 so the notation there is correct

Reply:
ahhh yep, that makes sense.

Reply:
updated my previous comment to note that the original notation was correct. It also caches the results